src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -15,13 +15,21 @@ $spacing: calc(1 * min(var(--section-seat-size), var(--section-seat-size-min)));
 .Section {
   background-attachment: fixed;
   background-position: center;
+  background-size: cover;
 
   position: relative;
   overflow: auto;
   height: 100%;
   padding: $spacing;
   min-width: fit-content;
 
+  &__background {
```

Comment:
this should go to ContainerWithBackground, i believe.

Reply:
fixed b396ea2